node_cli/schema/resource.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "resource",
+    "description": "health and manifest commands",
+    "sub_commands": [
+        {
+            "name": "show",
+            "description": "reshource show health and manifest command",
```

Comment:
Change description:
`Show resource health and manifest`

Reply:
Done